<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="DataPlot" name="plotArea" native="true"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionImport"/>
    <addaction name="actionImportGates"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuPlots">
    <property name="title">
     <string>&amp;Plots</string>
    </property>
    <addaction name="actionElevation"/>
    <addaction name="actionVerticalSpeed"/>
    <addaction name="actionHorizontalSpeed"/>
    <addaction name="actionTotalSpeed"/>
    <addaction name="actionDiveAngle"/>
    <addaction name="actionCurvature"/>
    <addaction name="actionGlideRatio"/>
   </widget>
   <widget class="QMenu" name="menuDomain">
    <property name="title">
     <string>&amp;Bottom</string>
    </property>
    <addaction name="actionTime"/>
    <addaction name="actionDistance2D"/>
    <addaction name="actionDistance3D"/>
   </widget>
   <widget class="QMenu" name="menuUnits">
    <property name="title">
     <string>&amp;Units</string>
    </property>
    <addaction name="actionMetric"/>
    <addaction name="actionImperial"/>
   </widget>
   <widget class="QMenu" name="menu_Tools">
    <property name="title">
     <string>&amp;Tools</string>
    </property>
    <addaction name="actionPan"/>
    <addaction name="actionZoom"/>
    <addaction name="actionMeasure"/>
   </widget>
   <widget class="QMenu" name="menuWindow">
    <property name="title">
     <string>Window</string>
    </property>
    <addaction name="actionShowLeftView"/>
    <addaction name="actionShowTopView"/>
    <addaction name="actionShowFrontView"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menuPlots"/>
   <addaction name="menuDomain"/>
   <addaction name="menuUnits"/>
   <addaction name="menu_Tools"/>
   <addaction name="menuWindow"/>
  </widget>
  <action name="actionImport">
   <property name="text">
    <string>Import &amp;Track...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+T</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>E&amp;xit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionElevation">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Elevation</string>
   </property>
   <property name="shortcut">
    <string>E</string>
   </property>
  </action>
  <action name="actionVerticalSpeed">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Vertical Speed</string>
   </property>
   <property name="shortcut">
    <string>V</string>
   </property>
  </action>
  <action name="actionHorizontalSpeed">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Horizontal Speed</string>
   </property>
   <property name="shortcut">
    <string>H</string>
   </property>
  </action>
  <action name="actionMetric">
   <property name="text">
    <string>&amp;Metric</string>
   </property>
  </action>
  <action name="actionImperial">
   <property name="text">
    <string>&amp;Imperial</string>
   </property>
  </action>
  <action name="actionTime">
   <property name="text">
    <string>&amp;Time</string>
   </property>
  </action>
  <action name="actionDistance2D">
   <property name="text">
    <string>Horizontal Distance</string>
   </property>
  </action>
  <action name="actionTotalSpeed">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Total &amp;Speed</string>
   </property>
   <property name="shortcut">
    <string>S</string>
   </property>
  </action>
  <action name="actionDiveAngle">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Dive Angle</string>
   </property>
   <property name="shortcut">
    <string>D</string>
   </property>
  </action>
  <action name="actionDistance3D">
   <property name="text">
    <string>Total Distance</string>
   </property>
  </action>
  <action name="actionImportGates">
   <property name="text">
    <string>Import &amp;Gates...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+G</string>
   </property>
  </action>
  <action name="actionGlideRatio">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Glide Ratio</string>
   </property>
   <property name="shortcut">
    <string>G</string>
   </property>
  </action>
  <action name="actionCurvature">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Curvature</string>
   </property>
   <property name="shortcut">
    <string>C</string>
   </property>
  </action>
  <action name="actionPan">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Pan</string>
   </property>
   <property name="shortcut">
    <string>P</string>
   </property>
  </action>
  <action name="actionZoom">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Zoom</string>
   </property>
   <property name="shortcut">
    <string>Z</string>
   </property>
  </action>
  <action name="actionMeasure">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Measure</string>
   </property>
   <property name="shortcut">
    <string>M</string>
   </property>
  </action>
  <action name="actionShowLeftView">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Side View</string>
   </property>
   <property name="shortcut">
    <string>Alt+1</string>
   </property>
  </action>
  <action name="actionShowTopView">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>T&amp;op View</string>
   </property>
   <property name="shortcut">
    <string>Alt+2</string>
   </property>
  </action>
  <action name="actionShowFrontView">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Front View</string>
   </property>
   <property name="shortcut">
    <string>Alt+3</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>DataPlot</class>
   <extends>QWidget</extends>
   <header>dataplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
